src/tools/qt-gui/src/main.cpp

```diff
@@ -12,15 +12,19 @@
 #include "confignode.hpp"
 #include "undomanager.hpp"
 #include "guibackend.hpp"
+#include "guisettings.hpp"
 
 int main(int argc, char* argv[])
 {
 	QApplication app(argc, argv);
+	//keynames are only split at unescaped slashes
+	app.setProperty("KEY_DELIMITER", QRegularExpression("(?<!\\\\)/"));
```

Comment:
Let's say there is a key "user/sw/app/foo/baloo". To create a the correct hierarchy in the model I need to know where to split the string into parts to create the according nodes (earlier the delimiter was "/" which will work fine in this case). But what if the keyname is "user/sw/app/foo\/baloo"?  We decided that escaped slashed should be part of the keyname, so the "/" delimiter will split the string at the wrong character. To avoid this, the regex is used to split the string at "/" but not at "\/". I do not see why keyGetBaseName() would help me to decide where to split the string?